dags/oss_know/libs/github/sync_profiles.py

```diff
@@ -1,198 +1,115 @@
-import time
 import datetime
+import redis
 import itertools
 import requests
-from opensearchpy import OpenSearch
+from oss_know.libs.util.base import get_opensearch_client
 from oss_know.libs.util.github_api import GithubAPI
 from oss_know.libs.base_dict.opensearch_index import OPENSEARCH_INDEX_GITHUB_PROFILE, OPENSEARCH_INDEX_CHECK_SYNC_DATA
+from oss_know.libs.base_dict.redis import STORAGE_HASH, STORAGE_IDS_SET, WORKING_HASH
 from oss_know.libs.util.log import logger
-from oss_know.libs.util.opensearch_api import OpensearchAPI
-
-
-class SyncGithubProfilesException(Exception):
-    def __init__(self, message, status):
-        super().__init__(message, status)
-        self.message = message
-        self.status = status
-
-
-def sync_github_profiles(github_tokens, opensearch_conn_info):
-    """Update existing GitHub profiles by id."""
-
+from opensearchpy import helpers as opensearch_helpers
+
+
+def get_redis_client(redis_client_infos):
```

Comment:
Consider moving the function to utils or base under lib, this is a pretty common function, should be provided to all modules, not only sync profile module.